weave/trace_server_bindings/remote_http_trace_server.py

```diff
@@ -161,6 +164,42 @@ def _flush_calls(
             self._flush_calls(batch[split_idx:], _should_update_batch_size=False)
             return
 
+        final_batch = []
+        if encoded_bytes > CLICKHOUSE_SINGLE_ROW_INSERT_BYTES_LIMIT:
+            # ensure that there isn't a single row that is too big
+            seen_bytes = 0
+            for i, item in enumerate(batch):
+                # below takes tens of miliseconds for the large payloads we skip
+                bytes_size = len(item.model_dump_json().encode("utf-8"))
+                seen_bytes += bytes_size
+                if bytes_size > CLICKHOUSE_SINGLE_ROW_INSERT_BYTES_LIMIT:
+                    mb = bytes_size / (1024**2)
+                    if isinstance(item.req, tsi.CallStartReq):
+                        id_, mode = (item.req.start.id, "start")
+                        item.req.start.inputs = ENTITY_TOO_LARGE_ERROR_PAYLOAD
```

Comment:
For sentinel: i think we could start with just a string, similar to our redaction string: `<EXCEED_LIMITS>`. Furthermore, for the `inputs` one, you should keep all input keys, and replace the large values with sentinel

Reply:
"you should keep all input keys" this is non trivial. What if the inputs have 5 keys each with 1MB payloads? We could iterate through all the values and keep track, and only replace the values once a running tally exceeds our limits... But this would totally fail in the case of thousands of input keys (admittedly not likely) or very large input keys, as the keys themselves would add to the data cap and we would still blow through our limit. @tssweeney

Reply:
I see. OK, well let's just do `{_weave: {error: ""}}` so we can put other keys inside there